spring-kafka/src/main/java/org/springframework/kafka/support/converter/ProjectingMessageConverter.java

```diff
@@ -61,11 +63,31 @@ public ProjectingMessageConverter() {
 	}
 
 	/**
-	 * Creates a new {@link ProjectingMessageConverter} using the given {@link ObjectMapper}.
+	 * Create a new {@link ProjectingMessageConverter} using the given {@link ObjectMapper}.
 	 * @param mapper must not be {@literal null}.
 	 */
 	public ProjectingMessageConverter(ObjectMapper mapper) {
+		this(mapper, new StringJsonMessageConverter());
+	}
+
+	/**
+	 * Create a new {@link ProjectingMessageConverter} using the given {@link ObjectMapper}.
+	 * @param delegate the delegate converter for outbound and non-interfaces.
+	 * @since 2.3
+	 */
+	public ProjectingMessageConverter(MessagingMessageConverter delegate) {
+		this(JacksonUtils.enhancedObjectMapper(), new StringJsonMessageConverter());
```

Comment:
How that happened that you don't use a `delegate` here at all?

Reply:
? I don't know what you mean.

Reply:
M-m-m. I mean that you don't use a `delegate` ctor argument in this method body. 
Not sure that it is intentional...

Reply:
Doh...